draft-ietf-core-dns-over-coap.md

```diff
@@ -229,20 +230,19 @@ Possible options to assure this could be manual configuration of a Uniform Resou
 or automatic configuration, e.g., using a CoRE resource directory
 {{-core-rd}}, DHCP or Router Advertisement options {{-dnr}}, or discovery of designated resolvers
 {{-ddr}}.
-Automatic configuration SHOULD only be done from a trusted source.
+Automatic configuration MUST only be done from a trusted source.
 
 ## Discovery by Resource Type
 For discovery of the DoC resource through a link mechanism that allows describing a resource type
 (e.g., the Resource Type attribute in {{-core-link-format}}), this document introduces the resource type "core.dns".
 It can be used to identify a generic DNS resolver that is available to the client.
 
 ## Discovery using SVCB Resource Records or DNR
-A DoC server can also be discovered using Service Binding (SVCB) Resource Records (RR) {{-svcb}} {{-svcb-dns}} or Discovery of Network-designated Resolvers (DNR)
-Service Parameters {{-dnr}}.
+A DoC server can also be discovered using Service Binding (SVCB) Resource Records (RR) {{-svcb}} {{-svcb-dns}} from a pre-established DNS service (e.g., an unencrypted local resolver) or Discovery of Network-designated Resolvers (DNR)
```

Comment:
weird wording "pre-estbalished DNS service". i suggest rather "A DoC server can also be discovered using Service Binding (SVCB) Resource Records (RR) {{-svcb}} {{-svcb-dns}} resolved via another DNS service (e.g., provided by an unencrypted local resolver)"